README.md

```diff
@@ -1 +1,52 @@
-# hello-rails-back-end
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Rails backend api only app
+
+> Create a Rails api only backend app that followes RESTfull design.
+
+## Built With
+
+- **Ruby**
+- **Rails**
+- **Postgresql**
+
+## React front-end link
+[react front-end](https://github.com/zmekonnen251/hello-react-front-end)
```

Comment:
- [ ] It's a requirement that you add the link for the backend repo in the readme of the front-end project. Kindly fix this